### PR TITLE
3730: switch ppms to use new staff.apps.va.gov url

### PR DESCRIPTION
**Pull request by kpethtel**
## Description of change
The PPMS service needs to connect to https://staff.apps.va.gov instead of https://veteran.apps.va.gov.

## Original issue(s)
https://app.zenhub.com/workspaces/va-mobile---backend-61d75d03a7505500147ec7f6/issues/department-of-veterans-affairs/va-mobile-app/3730

Devops PR: https://github.com/department-of-veterans-affairs/devops/pull/11876

## Things to know about this PR
This does involve changes to the settings file. I'm working with @RachalCassity on that. 

I will test this on the review instance once it's up. Unclear if that will need to wait until after the settings changes.

### CODE REVIEW

File: config/settings.yml [hunk 1]
```diff
@@ -1009,6 +1009,7 @@ directory:
 
 va_mobile:
   url: "https://veteran.apps.va.gov"
+  ppms_base_url: "https://staff.apps.va.gov/"
```
Comment (kpethtel) [inline config/settings.yml:1012]: I'm open to different naming conventions.

File: config/settings.yml [hunk 2]
```diff
@@ -1009,6 +1009,7 @@ directory:
 
 va_mobile:
   url: "https://veteran.apps.va.gov"
+  ppms_base_url: "https://staff.apps.va.gov"
```
Comment (kpethtel) [inline config/settings.yml:1012]: I'm open to renaming this something other than `ppms_base_url` if anyone has better ideas.

**Comment by LindseySaari** [comment]
Just want to note that you will need a corresponding PR in the devops repo for the settings changes here. e.g. https://github.com/department-of-veterans-affairs/devops/blob/master/ansible/deployment/config/vets-api/prod-settings.local.yml.j2

Additional reminder that the configmap in the manifests repo will also need to be updated for vets-api-server and sidekiq

File: config/settings.yml (hunk 2 above)
Comment (kpethtel) [reply config/settings.yml:1012]: That's a good question and I'll have to ask @simiadebowale for clarification on that. As you can see, we use a production URL directly above it as well. I believe this only impacts local development and tests. There seem to be a few places in this file where we use production urls. Perhaps they should all be changed to staging and the VCR cassettes updated to match.

**Review by simiadebowale — APPROVED** [review]

**Pull request merged**